pruebas/scripts/seeker_listener.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
I think we should add a module/script docstring here, to briefly explain what this tries to do.
https://realpython.com/documenting-python-code/#script-docstrings 
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

```suggestion
#!/usr/bin/env python

""" this does something

asdklfjs dkjfsdl jfs df flkjsa flsd f
aslkdjfskdjf
```

Reply:
I'll resolve it later

Reply:
OK